tests/dummy/app/controllers/sparkline.js

```diff
@@ -0,0 +1,71 @@
+import Ember from 'ember';
+import ColumnDefinition from 'ember-table/models/column-definition';
+
+export default Ember.Controller.extend({
+  tableColumns: Ember.computed(function() {
+    var name = ColumnDefinition.create({
```

Comment:
out of scope, but I would create an array of column definitions, and only after use `ColumnDefintion.create`

```js
var columnDefintions = [{
    ...
}, {
    ...
}]

return columnDefintions.map( function( columnDefintion ) {
   return ColumnDefinition.create( columnDefintion );
});
```